tests/acceptance/features/lib/FilesPageElement/SharingDialog.php

```diff
@@ -418,13 +429,21 @@ public function setSharingPermissions(
 	 * @return void
 	 */
 	public function checkSharingPermissions(
+		$userOrGroup,
 		$shareReceiverName,
 		$permissions,
 		Session $session
 	) {
-		$xpathLocator = \sprintf(
-			$this->permissionsFieldByUserName, $shareReceiverName
-		);
+		if ($userOrGroup == "group") {
+			$permissionFieldByGroup = $this->permissionsFieldByGroupName;
+			$xpathLocator = \sprintf(
+				$permissionFieldByGroup, $shareReceiverName
```

Comment:
IMO we do not need the local variable at all with this code now.
```suggestion
				$this->permissionsFieldByGroupName, $shareReceiverName
```